airflow/www/templates/airflow/graph.html

```diff
@@ -362,6 +362,15 @@
 
 </script>
 <script src="{{ url_for_asset('graph.js') }}"></script>
+<script src="{{ url_for_asset('index.js') }}"></script>
```

Comment:
How does this play with the import of ` export { initRefreshButton, update_nodes_states } from './graph';`  inside index.js? Is the include of graph.js on the previous line the same file?